### DISCUSSION TITLE
Welcome to webauthn4j-spring-security Discussions!

### DISCUSSION THREAD

**Discussion by ynojima** [webauthn4j/webauthn4j-spring-security#495] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TGITS** [comment]
Hello,
I am a French developer.
One of my current tasks is to implement a WebAuthn proof-of-concept application using Angular for the front and the Spring Boot stack for the back. Hopefully this POC will validate the use of WebAuthn for use in production.
I plan to test and use webauthn4j and webauthn-spring-security to reach this aim, and in the process contribute to this project if I can.
